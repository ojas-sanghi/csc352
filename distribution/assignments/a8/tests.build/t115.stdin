15 2 
22 15 

19 7 



22 3 
10 5 

13 
19 11 
18 


