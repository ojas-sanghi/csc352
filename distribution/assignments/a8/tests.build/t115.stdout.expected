((-2((-3(-5-))7(-10((-11-)13-))))15((-15((-18-)19(-19-)))22(-22-)))
Insert 10
((-2((-3(-5-))7(-10(((-10-)11-)13-))))15((-15((-18-)19(-19-)))22(-22-)))
